resources/definitions/vertex_delta_k8800.def.json

```diff
@@ -1,65 +1,62 @@
-            "default": 0
-        },
-        "machine_head_shape_max_x": {
-            "default": 0
-        },
-        "machine_head_shape_max_y": {
-            "default": 0
-        },
-        "machine_nozzle_gantry_distance": {
-            "default": 0
-        },
-        "machine_nozzle_offset_x_1": {
-            "default": 0
-        },
-        "machine_nozzle_offset_y_1": {
-            "default": 0
-        },
-        "machine_gcode_flavor": {
-            "default": "RepRap"
-        },
-        "machine_start_gcode": {
-            "default": "; Vertex Delta Start Gcode\nM0 Is my nozzle clean?\nM400\nG28 ; Home extruder\nM106 S128 ; Start fan\nM104 T0 R130 ; Set cold nozzle\nM109 T0 R130 ; Wait for cold nozzle\nM117 Leveling bed...\nG29 ; Level Bed\nG1 X0 Y100 Z1 F2000\nG92 Z0.9 ; Set Z position (SET Z OFFSET HERE -> 1 - OFFSET)\nM107 ; Stop fan\nG90 ; Absolute positioning\nM82 ; Extruder in absolute mode\nM104 T0 S{material_print_temperature}\nG92 E0 ; Reset extruder position\nM109 T0 S{material_print_temperature}\nM117 Priming nozzle...\nM83\nG1 E20 F100 ; purge/prime nozzle\nM82\nG92 E0 ; Reset extruder position\nG4 S3 ; Wait 3 seconds\nG1 Z5 F2000\nM117 Vertex Delta printing"
-        },
-        "machine_end_gcode": {
-            "default": "; Vertex Delta end code\nM107 ; Turn off fan\nG91 ; Relative positioning\nT0\nG1 E-1 F1500; Reduce filament pressure\nM104 T0 S0\nG90 ; Absolute positioning\nG92 E0 ; Reset extruder position\nM300 S4000 P500\nM300 S3000 P500\nM300 S2000 P800\nG28\nM84 ; Turn steppers off"
-        }
+  "id": "K8800",
+  "name": "Vertex Delta K8800",
+  "version": 2,
+  "inherits": "fdmprinter",
+  "metadata": {
+    "manufacturer": "Velleman nv",
+	"file_formats": "text/x-gcode",
+    "visible": true,
+    "author": "Velleman"
+  },
+  "overrides": {
+    "machine_width": {
+      "default_value": 200
     },
-    "categories": {
-        "material": {
-            "settings": {
-                "material_bed_temperature": {
-                    "visible": true
-                }
-            }
-        }
+    "machine_height": {
+      "default_value": 225
+    },
+    "machine_depth": {
+      "default_value": 200
+    },
+    "machine_center_is_zero": {
+      "default_value": true
+    },
+    "machine_shape": {
+      "default_value": "elliptic"
+    },
+    "machine_nozzle_size": {
+      "default_value": 0.35
+    },
+    "machine_head_shape_min_x": {
+      "default_value": 0
+    },
+    "machine_head_shape_min_y": {
+      "default_value": 0
+    },
+    "machine_head_shape_max_x": {
+      "default_value": 0
+    },
+    "machine_head_shape_max_y": {
+      "default_value": 0
+    },
+    "machine_nozzle_gantry_distance": {
+      "default_value": 0
+    },
+    "machine_nozzle_offset_x_1": {
+      "default_value": 0
+    },
+    "machine_nozzle_offset_y_1": {
+      "default_value": 0
+    },
+    "machine_gcode_flavor": {
+      "default_value": "RepRap"
```

Comment:
This value should be one of the following:
- `"RepRap (Marlin/Sprinter)"`
- `"RepRap (Volumetric)"`
- `"RepRap (RepRap)"`